Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM cyberark/ubuntu-ruby-fips:latest
+FROM registry2.itci.conjur.net/cyberark/ubuntu-ruby-fips:20.04-b201fa9
```

Comment:
Should be switched back after base image release

Reply:
Will rollback before push to master